### PR TITLE
SEO-173296 WPF Chart SeriesTypes

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @MallikaRK ,

Have renamed image folder, changed and image alt.

Regards,
Yvone.

### CODE REVIEW

File: wpf/Charts/SeriesTypes/Financial.md [hunk 1]
```diff
@@ -237,7 +237,7 @@ series.ComparisonMode = Syncfusion.UI.Xaml.Charts.FinancialPrice.High;
 
 {% endtabs %}   
 
-![Comparing High Value of WPF Candle Chart](Series_images/wpf-cadle-chart-compare-high-value.png)
+![Comparing high value of WPF Candle Chart](series_images/wpf-candle-chart-compare-high-value.png)
```
Comment (MallikaRK) [inline wpf/Charts/SeriesTypes/Financial.md:240]: You have changed the image name here. But you missed to edit the image name in series_images folder. Recheck again.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
Commented

File: wpf/Charts/SeriesTypes/Financial.md (hunk 1 above)
Comment (Yvone-Atieno) [reply wpf/Charts/SeriesTypes/Financial.md:240]: renamed

**Review by MallikaRK — APPROVED** [review]
Approved

**Review by Girija-Rajendran — APPROVED** [review]

**Pull request merged**